src/data/footerLinks.json

```diff
@@ -49,8 +49,7 @@
                     "to": "https://www.youtube.com/c/Helpshift"
                 },
                 {
-                    "label": "Contact Us",
-                    "to": "https://www.helpshift.com/company/contact-us/"
+                    "html": "<span class=\"footer__link-item cursor-pointer\" onClick='Helpshift(\"open\")'>Report an issue</span>"
```

Comment:
Optional and low-priority: Did we finally get our GA integration working on dev docs? If yes, consider adding an analytics event to track clicks. Alternatively, we can track it based on metadata (source page URL) on the admin dashboard.